app/src/app/data.service.ts

```diff
@@ -95,6 +96,7 @@ export class DataService {
         this.slittedBandPlanning = this.split(this.band.planning, '.');
```

Comment:
Would be better to move the splitting logic into the class that consumes the data, rather than in the data service.